### ISSUE TITLE
Package cert exprired 

### ISSUE THREAD

**Issue by clint5047** [ygini/Hello-IT#32]
The cert in 1.2.9 has expired, leaving it uninstallable by Casper.

![screen shot 2017-07-10 at 2 22 03 pm_thumb](https://user-images.githubusercontent.com/30055287/28033477-65afe2c4-657c-11e7-8e9b-d20bdbfc966e.png)

**Comment by WardsParadox** [comment]
If you want, you can strip the cert (I am not sure if JAMF lets you install unsigned packages) and it's generally a bad idea, but if you need to get it out to your clients it might be the way you have to go:
https://managingosx.wordpress.com/2012/03/24/fixing-packages-with-expired-signatures/
``` sh
pkgutil --expand SomeFlat.pkg /tmp/SomeFlat.pkg
pkgutil --flatten /tmp/SomeFlat.pkg SomeFlatFixed.pkg
```

**Comment by clint5047** [comment]
Thanks! That did the trick!

**Comment by ygini** [comment]
This should now be fixed with release 1.2.11. Sorry for the delay, MacAdmins PSU time isn't the best time to publish a new software release.
